peerplays/cli/asset.py

```diff
@@ -0,0 +1,44 @@
+from .decorators import (
+    onlineChain,
+)
+
+try:
+    import click
```

Comment:
You don't need this try, catch. click is a dependency that is installed together with the library